src/main/java/com/epam/reportportal/junit/ParallelRunningHandler.java

```diff
@@ -438,31 +432,31 @@ protected FinishTestItemRQ buildFinishStepRq(FrameworkMethod method, String stat
 		return parameters.isEmpty() ? null : parameters;
 	}
 
-	protected Optional<TestCaseIdEntry> getTestCaseId(FrameworkMethod method, Object runner, String codeRef) {
-		if (!(method.isStatic() || isIgnored(method))) {
-			Object target = LifecycleHooks.getTargetForRunner(runner);
-			return ofNullable(method.getMethod().getDeclaredAnnotation(TestCaseId.class)).flatMap(annotation -> {
-				if (annotation.parametrized() && target instanceof ArtifactParams) {
-					return getParameterizedTestCaseId(target, codeRef);
-				} else {
-					return Optional.of(annotation.value()).map(value -> new TestCaseIdEntry(value, value.hashCode()));
-				}
-			});
+	protected TestCaseIdEntry getTestCaseId(FrameworkMethod method, Object runner, String codeRef) {
```

Comment:
Do we need TestCaseIdEntry class if we don't use hash anymore?

Reply:
It's not necessary, but more concise to have wrapper for such entities. 
We can either use value from the annotation or generate it, so I suppose it won't hurt us to have this wrapper as an abstactioj